yt_dlp/extractor/wistia.py

```diff
@@ -145,17 +152,74 @@ class WistiaIE(WistiaBaseIE):
         'only_matching': True,
     }]
 
-    # https://wistia.com/support/embed-and-share/video-on-your-website
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        embed_config = self._download_embed_config('medias', video_id, url)
+        return self._extract_media(embed_config)
+
+
+class WistiaEmbedIE(WistiaBaseIE):
```

Comment:
Why does this need to be a separate extractor?

Reply:
I've split it out now.

Some of the embed code was tightly shared between the extractors previously hence I merged it all.